tests/test_options.py

```diff
@@ -68,14 +68,14 @@ def test_width_height(images: List[Path]):
     # https://github.com/jgraph/drawio-desktop/issues/254
```

Comment:
Could you remove this comment and mention this  URL in the commit message body? This is what has been fixed so we are adjusting the tests.

Reply:
Done :heavy_check_mark: